ci-operator/step-registry/upi/conf/vsphere/ova/upi-conf-vsphere-ova-commands.sh

```diff
@@ -62,21 +50,68 @@ for i in "${!DATACENTERS[@]}"; do
     CLUSTER=$(echo -n ${CLUSTERS[$i]} |  tr -d '\n')
     RESOURCE_POOL="/$DATACENTER/host/$CLUSTER/Resources"
     export GOVC_RESOURCE_POOL=$RESOURCE_POOL
+
+    OVA_NETWORK=""
+    NETWORKS=($(govc find -i network -name $LEASED_RESOURCE))
+
+    # Search the found networks for a network that exists in the target cluster.
+    echo "$(date -u --rfc-3339=seconds) - Validating network configuration... ${#NETWORKS[@]}"
+    if [ "${#NETWORKS[@]}" -gt 1 ]; then
+        echo "$(date -u --rfc-3339=seconds) - Detected multiple matching networks.  Searching for valid network target."
+        for NET in ${NETWORKS[@]}; do
+            case "${NET}" in
+                DistributedVirtualPortgroup*)
```

Comment:
what is the advantage of case over if/else(i.e. if [[ "${NET} == *DistributedVirtualPortgroup* ]])?

Reply:
Currently there are a few others that may return.  For our environment, we are currently only getting this one back, but I wanted to make sure I logged if we ever got something else back.  Since there is currently only one, I'm happy to change to If.  The * was easier for me to do startwith, but gotta make sure its a startswith check not equals.

Reply:
no need to change it, that's fine, I was just wondering.